e2e/fuzzing_test.go

```diff
@@ -60,6 +61,38 @@ var _ = Describe("E2E TEST:"+checks.CheckFuzzing, func() {
 			Expect(repoClient.Close()).Should(BeNil())
 			Expect(ossFuzzRepoClient.Close()).Should(BeNil())
 		})
+		It("Should return use of OSS-Fuzz - GitLab", func() {
```

Comment:
Does it matter if this runs with the GitHub integration or the GitLab ones, or both? 
Since this uses the ossfuzz client, I don't think so, but I know some of the gitlab repo init stuff reaches out to the network.

Feel free to resolve, I think the answer is no, but wanted to post.

Reply:
This test shouldn't make a request to either GitHub or GitLab (OSS Fuzz results are fetched from the GCS bucket). But it does connect to the network, and is a GitLab-specific test, so I opted to leave it as an E2E test with the ` - GitLab` marker.